Sources/Commands/SwiftPackageTool.swift

```diff
@@ -908,7 +914,7 @@ extension SwiftPackageTool {
         @OptionGroup()
         var swiftOptions: SwiftToolOptions
 
-        @Argument()
+        @Argument(help: "generate-bash-script | generate-zsh-script |\ngenerate-fish-script | list-dependencies | list-executables")
```

Comment:
Can the argument parser library enumerate the enum options instead of having to explicitly spell them like this? cc @natecook1000?